Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'kaminari-activerecord'
 gem 'pg'
 gem 'ransack'
 gem 'with_advisory_lock'
+gem 'paper_trail'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `paper_trail` should appear before `with_advisory_lock`.